cartridges/int_extend_sfra/cartridge/scripts/jobHelpers.js

```diff
@@ -32,8 +32,54 @@ function getContractLoggerModel (order) {
     return result;
 }
 
+var refundStatus = {
+    SUCCESS: 'SUCCESS',
+    REJECT: 'REJECT',
+    ERROR: 'ERROR',
+    REJECT_AND_ERROR: 'REJECT_AND_ERROR'
+}
+
+function getRefundStatus (order) {
+    var error = false;
+    var reject = false;
+
+    for (var i = 0; i < order.productLineItems.length; i++) {
+        var pLi = order.productLineItems[i];
+
+        if (!pLi.custom.extendRefundStatuses) {
+            continue;
+        }
+
+        var extendRefundStatuses = JSON.parse(pLi.custom.extendRefundStatuses);
+        var statuses = Object.keys(extendRefundStatuses);
+
+        for (var j = 0; j < statuses.length; j++) {
+            var contract = statuses[j];
+            var status = extendRefundStatuses[contract];
+
+            if (status === refundStatus.ERROR) {
+                error = true;
+            } else if (status === refundStatus.REJECT) {
+                reject = true;
+            }
+        }
+    }
+
+    if (error && reject) {
```

Comment:
please do not duplicate the code, use helpers

Reply:
This is not a duplication, but another cartridge with the same code.